BHoM_Adapter/AdapterActions/_PushMethods/CRUDDispatchers/UpdateOnly.cs

```diff
@@ -57,7 +57,7 @@ public abstract partial class BHoMAdapter
                 var dependencyObjects = Engine.Adapter.Query.GetDependencyObjects<T>(newObjects, dependencyTypes, tag); //first-level dependencies
 
                 foreach (var depObj in dependencyObjects)
-                    if (!FullCRUD(depObj.Value as dynamic, PushType.FullCRUD, tag, actionConfig))
+                    if (!FullCRUD(depObj.Value as dynamic, PushType.FullPush, tag, actionConfig))
```

Comment:
Shouldn't we change `FullCRUD` method name to `FullPush` in order to keep consistent?

Reply:
Method name stays as only enums are exposed to the users which shouldn't necessarily need to know of the CRUD methodology